### DISCUSSION TITLE
Can renovate be configured to close a PR if a release is deleted?

### DISCUSSION THREAD

**Discussion by djfinnoy** [renovatebot/renovate#14811] [Request Help] [answered]
I'm considering renovate for the next iteration of a GitOps CICD pipeline.
I've skimmed through the docs, but I'm not sure whether what I have in mind is feasible.

Assume there are two environments, `dev` and `prod`.
There are many bespoke microservice repositories, and one (github) repo with Helm charts that is monitored by a kubernetes GitOps tool (ArgoCD).

I would like my developers to deploy new versions of their microservices to `prod` by creating releases, and to `dev` by creating pre-releases. Once a pre-release becomes a proper release, it is rolled out in both `dev` and `prod`.

When there is a new release in a microservice repo, renovate would create a PR in the Helm repo where it modifies image tags. The rest of the required automation will be handled with GitHub actions and ArgoCD.

The scenario I'm not sure if renovate can handle is when a pre-release is discarded.
Can renovate be configured to close a PR if the release that triggered it disappears?

**Comment by rarkins** [comment, marked as answer]
Renovate always closes its own PRs if an update is no longer required, for any reason.

**Reply by djfinnoy** [reply]
Excellent, thank you.
